cfme/automate/dialog_element.py

```diff
@@ -99,7 +100,9 @@ def add_another_element(self, element):
         view = navigate_to(self, 'Edit')
         view.element_tree.click_path(*self.tree_path[1:])
         view.plus_btn.item_select("Add a new Element to this Box")
-        view.fill(element)
+        view.fill(element.get('element_information'))
+        if element.get('options') is not None:
```

Comment:
I believe this is a redundant check against the value being None - a fill with a None value gets ignored by default.

Reply:
This is code for version 5.8 , where views are not nested so .. it raises error if value is None . 
SO need to do a check here

Reply:
Noted, thanks.

That being the case, I think your explanation needs to be an in-line comment here. (required)

Reply:
Done